tests/unittests/operators/executor_tests.py

```diff
@@ -30,12 +27,11 @@ def execute(self, ctx):
         return {"message": ctx.params.get("message", None)}
 
 
-# Force registration of the operator for testing
-registry._EXTRA_OPERATORS.append(EchoOperator(_builtin=True))
-
-
 @pytest.mark.asyncio
-async def test_execute_or_delegate_operator():
+@patch("fiftyone.operators.registry.OperatorRegistry.list_operators")
```

Comment:
Interesting! I just used this in teams but as `unittest.mock.patch()`. 

Is there a way to use the real `list_operators`? Thats a big part of whats being tested here. Although `_EXTRA_OPERATORS` isn't a better solution.

Reply:
I thought about it! We can achieve the same effect by mocking out `fopc.build_plugin_contexts` but I think it's cleaner to mock out `list_operators`.

I can add a unit test for `list_operators` too so that it is also covered by tests.